js/src/components/google-combo-account-card/connected-google-combo-account-card.js

```diff
@@ -22,14 +23,19 @@ const ConnectedGoogleComboAccountCard = () => {
 	}
 
 	return (
-		<AccountCard
-			appearance={ APPEARANCE.GOOGLE }
-			alignIcon="top"
-			className="gla-google-combo-account-card--connected"
-			description={ text || <AccountDetails /> }
-			helper={ subText }
-			indicator={ <Indicator showSpinner={ Boolean( creatingWhich ) } /> }
-		/>
+		<div className="gla-google-combo-account-card gla-account-card">
```

Comment:
Removed `gla-google-combo-account-card` as it is not being used anywhere.

Reply:
@eason9487 I've removed the unused class name. I've kept the surrounding div though, so that the cards are not directly children of `gla-vertical-gap-layout__large` where there'll be a gap between the cards.